.github/workflows/conformance-aks-v1.11.yaml

```diff
@@ -196,12 +197,10 @@ jobs:
           target_url: ${{ env.check_url }}
 
       - name: Install Cilium CLI
-        run: |
-          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
-          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
-          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
-          rm cilium-linux-amd64.tar.gz{,.sha256sum}
-          cilium version
+        uses: cilium/cilium-cli@d3c7168e26e2897bae563eb41aa9692db3fa0962
```

Comment:
If I'm not mistaken, this will make dependanbot to update cilium/cilium-cli SHA on a weekly basis, or even revert to the last stable release.

Reply:
Isn't the SHA only for the action? I think the version of cilium-cli binary being installed is still the one specified in `cilium_cli_version` (or `cilium_cli_ci_version`).

Reply:
Ah, I guess this is what you meant :sweat_smile: the GH action is not yet part of a released version.

I plan to tag v0.14.1 later today, so we could use that SHA then.

Reply:
To avoid frequent updates, can't we stick to the `main`? I know about security concerns, but in this case we are controlling the `cilium/cilium-cli` repo.

Reply:
Changed to v0.14.1. Eagerly waiting for the release (a GHA cannot be used from unrealeased tag).

Reply:
https://github.com/cilium/cilium-cli/releases/tag/v0.14.1

Reply:
Thanks! Updated. PTAL.